jahia-cloud-env-update/update-to-2.1.0/update.yml

```
@@ -27,3 +27,10 @@ actions:
         systemctl daemon-reload
         exit $?
       user: root
+
+  healthcheckOnHaproxy:
+    - cmd[bl]: |-
+        cp /etc/haproxy/haproxy.cfg.d/00-global.cfg /root/00-global.cfg.bak
+        sed -ie '/\!tools #HTTP_AUTH_BASIC/a acl healthcheck path /healthcheck\n    http-request allow if healthcheck' /etc/haproxy/haproxy.cfg.d/00-global.cfg |grep -C5 START
+        service haproxy reload
```

I don't understand the `grep` and also it's could be more simple with better result:
```suggestion
        sed -i '73i\    acl healthcheck path /healthcheck\n    http-request allow if healthcheck\n' /etc/haproxy/haproxy.cfg.d/00-global.cfg
        service haproxy reload
```

---

Sorry, the grep is useless indeed (used for tests, forgot to remove it...)
I think specifying the line is dangerous. If for any reason a blank line has been added in the file for test/debug purposes, this won't place it where we want.